content/en/apps/guides/data/analytics/testing-dbt-models.md

```diff
@@ -0,0 +1,306 @@
+
+
+### Data tests
+
+Data tests can be further divided into two types:
+- [**generic tests**](https://docs.getdbt.com/docs/build/data-tests#generic-data-tests): These are foundational tests provided by dbt core, focusing on basic schema validation and source freshness. dbt core provides four built-in generic tests that are essential for data modeling and ensuring data integrity. Generic tests can accept [additional test configurations](https://docs.getdbt.com/reference/data-test-configs).
+It is also possible to define your own [custom generic tests](https://docs.getdbt.com/best-practices/writing-custom-generic-tests).
+
+- [**singular tests**](https://docs.getdbt.com/docs/build/data-tests#singular-data-tests): These are written in a SQL file with a query that returns records that fail the test. This type of test is straightforward and focuses on specific conditions or rules that data must meet.
+
+### Unit tests
+
+Unit tests are essential for validating complex SQL logic and transformations in dbt models. These tests are especially valuable when working with intricate SQL expressions. They help catch errors before deploying changes, ensuring the logic behaves as expected, particularly in critical models or when handling edge cases.
+
+For more details on formatting unit tests, refer to the [official dbt documentation](https://docs.getdbt.com/reference/resource-properties/unit-tests).
+
+## Guidelines for CHT Pipeline tests
+To ensure data integrity and the reliability of the dbt models in the CHT Pipeline, it is essential to follow these testing guidelines:
+
+- **Basic generic tests** for all models:
+Every model should have generic tests to enforce critical constraints and relationships. Use dbt core build in generic tests.
+
+- **Singular or custom generic data tests** for aggregation models:
+For models that perform data aggregation, it is crucial to include singular data tests or custom generic data tests to ensure that the aggregated data meets the expected criteria. These tests help verify that:
+  - Data is accurately aggregated according to the defined logic.
+  - The results align with business expectations and requirements.
+  - Custom data tests are particularly important for aggregation models, as errors in these models can lead to significant discrepancies in reports and analyses.
+
+- **Unit tests** for complex logic:
+Unit tests are not strictly required but are highly recommended, especially for models with complex transformation logic. Examples of when to use unit tests include:
+  - Complex SQL Logic: Such as regex, date math, window functions, or extensive `CASE WHEN` statements.
+  - Custom calculations: When creating functions or applying unique data processing logic.
+  - Edge cases: To handle scenarios that are not typically found in actual data but may arise unexpectedly.
+
+## Writing CHT Pipeline tests
```

Comment:
The first three lines here feel repetitive because this is already mentioned in the guidelines section above. I recommend removing them altogether.